src/index.html

```diff
@@ -4,9 +4,400 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="nav">
+      <div class="nav__container">
+        <a class="nav__logo" href="index.html"><img src="./images/logo.png" alt="Moyo_logo"></a>
+        <nav>
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a class="nav__list-link is-active"
+                 href="#apple"
+                 data-qa="nav-hover"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link" href="#samsung">
+                Samsung
+              </a>
+            </li>
+            <li class="nav__list-item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="nav__list-item">
```